File: src/test/java/android/test/sample/SampleAndroidSuite.java
package android.test.sample;

import org.junit.experimental.categories.Categories;
import org.junit.runner.RunWith;

import static org.junit.runners.Suite.SuiteClasses;

@SuiteClasses({
    SampleListAndroidTest.class,
    SampleDetailsAndroidTest.class
})
@RunWith(Categories.class)
public final class SampleAndroidSuite {
}

File: src/test/java/android/test/sample/SampleAndroidSuite.kt
package android.test.sample

import org.junit.experimental.categories.Categories
import org.junit.runner.RunWith
import org.junit.runners.Suite.SuiteClasses

@SuiteClasses(
    SampleListAndroidTest::class,
    SampleDetailsAndroidTest::class
)
@RunWith(Categories::class)
class SampleAndroidSuite